build/build.jl

```diff
@@ -0,0 +1,9 @@
+using PackageCompiler, Libdl
+
+PackageCompiler.create_sysimage(Symbol[:Diffusion];
+                                project=".",
```

Comment:
```suggestion
                                project=joinpath(@__DIR__, ".."),
```

Reply:
Shouldn't it then rather be 
```julia
project=joinpath(@__DIR__, ".")
```
?
```julia
project="." 
```
refers to the current directory but your suggestion to the one above, doesn't it? 

This build script (`build/build.jl `) is invoked from Makefile, i.e. from the directory where the Makefile is, i.e. from the top level directory of the repo.

Reply:
`@__DIR__` is the directory of the file, not the working directory.